plugins/services/src/js/pages/task-details/TaskLogsTab.js

```diff
@@ -21,7 +21,8 @@ const METHODS_TO_BIND = [
   'handleItemSelection'
 ];
 
-const PAGE_ENTRY_COUNT = 200;
+// Number of lines (entries) we asses to be a page
+const PAGE_ENTRY_COUNT = 400;
```

Comment:
This is an arbitrary number that describes how many lines that goes into one page. I landed on this number by trial and error and doubled it this time because we are no longer requesting "two half" pages initially, just one and 400 lines seemed to be a good number.